tools/README.md

```diff
@@ -0,0 +1,33 @@
+# legalcode tools
+
+Tools for working with creativecommons.org's legal code content.
+
+
+## Develpment
+
+Prior to submitting a pull request, please validate the style/syntax (from
+within this tools directory):
+1. [Black](https://black.readthedocs.io/en/stable/): The uncompromising code
+   formatter
+    ```shell
+    black ./
+    ```
+2. [Flake8](http://flake8.pycqa.org/en/latest/): Your Tool For Style Guide
+   Enforcement
+    ```shell
+    flake8 ./
+    ```
```

Comment:
Is it worth creating a lint script that runs these `black` and `flake8` commands together instead of having to run them separately?